### ISSUE TITLE
cannot login

### ISSUE THREAD

**Issue by sdahlbac** [amalfra/docker-hub#20]
On osx

>docker-hub version
docker-hub 4.0.0

```
docker login
Login with your Docker ID to push and pull images from Docker Hub. If you don't have a Docker ID, head over to https://hub.docker.com to create one.
<redacted>
Login Succeeded
```

```

 docker-hub repos --orgname <notrelevant>
##################################
 Found 5 results. On page 1 of 1
##################################
+---------------+-------------------+--------------+--------------+------------------+
| Name          | Repository Type   |   Star count |   Pull count | Last updated     |
+===============+===================+==============+==============+==================+
| apple-bloom   |                   |            0 |         2508 | 2016-10-20 09:20 |
+---------------+-------------------+--------------+--------------+------------------+
| ubuntu        | image             |            0 |          446 | 2018-08-14 06:56 |
+---------------+-------------------+--------------+--------------+------------------+
| centos        | image             |            0 |          231 | 2017-06-22 13:48 |
+---------------+-------------------+--------------+--------------+------------------+
| wine-electron | image             |            0 |         3488 | 2021-12-06 17:09 |
+---------------+-------------------+--------------+--------------+------------------+
| agent         | image             |            0 |            6 | 2021-10-01 13:32 |
+---------------+-------------------+--------------+--------------+------------------+

**Tip: You are not authenticated with Docker Hub. Hence only public resources will be fetched. Try authenticating using `docker login` or `docker-hub login` command to see more.**
```

```
 docker-hub login
Traceback (most recent call last):
  File "/opt/homebrew/bin/docker-hub", line 8, in <module>
    sys.exit(main())
  File "/opt/homebrew/lib/python3.9/site-packages/src/__main__.py", line 78, in main
    command.run(docker_hub_client, args)
TypeError: run() takes 1 positional argument but 2 were given
```

**Comment by amalfra** [comment]
@sdahlbac This is fixed in v4.0.1 which got released now. Please try that and feel free to reopen it if it still exists.

**Comment by sdahlbac** [comment]
Confirmed working, and thanks for the instant fix!
